cypress/cypress/e2e/AE_Tests.cy.js

```diff
@@ -5,8 +5,8 @@ describe("AE Tests", () => {
   const username = "name";
   const address1 = "ABC";
   const address2 ="CBD";
-  const testsNeedingAccount = ["AE_TestCase2"];
-
+  const testsNeedingAccount = ["AE_TestCase2", "AE_TestCase5"];
```

Comment:
Don't forget to do a final pull request for the main branch before merging. There is one more test case with this const change.